x-pack/solutions/security/plugins/security_solution/public/data_view_manager/components/data_view_picker/index.tsx

```diff
@@ -84,8 +85,18 @@ export const DataViewPicker = memo(({ scope, onClosePopover, disabled }: DataVie
           },
         });
       }
+
+      if (isExploreSourcerer) {
```

Comment:
This makes sure that the selected data view is preserved on the explore pages on refresh